archive-redirect.js

```diff
@@ -0,0 +1,83 @@
+var URL_ARCHIVED_WEBMAKER = 'https://old.webmaker.org';
+var URL_EVENTS_WEBMAKER = 'https://events.webmaker.org';
+var URL_TEACH_SITE = 'https://teach.mozilla.org';
+
+exports.redirectMap = [{
+  to: URL_ARCHIVED_WEBMAKER + '/tools',
+  from: [
+    '/tools/x-ray-goggles',
+    '/tools/x-ray-goggles/install'
+  ]
+}, {
+  // Switch to SSL after Bug 883370 lands.
+  to: 'http://blog.webmaker.org',
+  from: [
+    '/news'
+  ]
+}, {
+  to: URL_ARCHIVED_WEBMAKER + '/getinvolved',
+  from: [
+    '/build',
+    '/get-involved'
+  ]
+}, {
+  to: URL_EVENTS_WEBMAKER + '/#!/event-guides',
+  from: [
+    '/guides',
+    '/event-guides'
+  ]
+}, {
+  to: 'https://support.mozilla.org/kb/translate-webmaker',
+  from: [
+    '/translate'
+  ]
+}, {
+  to: URL_EVENTS_WEBMAKER,
+  from: [
+    '/events'
+  ]
+}, {
+  to: URL_EVENTS_WEBMAKER + '/#!/events/',
+  from: [
+    '/events/{event}'
+  ]
+}, {
+  to: URL_ARCHIVED_WEBMAKER + '/explore',
+  from: [
+    '/gallery'
+  ]
+}, {
+  to: 'https://hivelearningnetworks.org/',
+  from: [
+    '/_hive'
+  ]
+}, {
+  to: URL_TEACH_SITE + '/events',
+  from: [
+    // Previous from that redirected to /party
+    '/partners',
+    '/party',
+    '/makerparty'
+  ]
+}, {
+  to: URL_TEACH_SITE + '/teach-like-mozilla/web-literacy',
+  from: [
+    // Previous from that redirected to /resources
+    '/network',
+    '/kits',
+    '/kit-prototypes',
+    '/starter-makes',
+    '/resources',
+    '/resources/{param*}',
+    '/literacy',
+    '/literacy/{param*}'
+  ]
+}, {
+  to: URL_TEACH_SITE,
+  from: [
+    // Previous from that redirected to /mentor
+    '/community',
+    '/connect',
+    '/mentor'
+  ]
+}];
```

Comment:
I just [updated this PR](https://github.com/mozilla/webmaker-browser/commit/8987c7e4b61b1c16ee811d590e21508b2af4ddcd) after I saw https://github.com/mozilla/webmaker.org/issues/1423#issue-95302410 

> https://webmaker.org/en-US/tools should redirect to https://teach.mozilla.org/tools/ (this will cover code.org I believe)

@jbuck should I submit a PR against [the `old` branch on webmaker.org](https://github.com/mozilla/webmaker.org/tree/old) for this as well?